wrappers/golang/internal/build-libs.go

```diff
@@ -70,6 +73,8 @@ func main() {
 		DevMode:          "OFF",
 		CudaBackend:      "OFF",
 		MetalBackend:     "OFF",
+		Pqc:              "OFF",
+		CudaPqcBackend:   "OFF",
```

Comment:
Not sure that this flag is necessary since it is overridden internally to always be ON if pqc is enabled